ci-operator/config/redhat-developer/rhdh/.claude/skills/rhdh-ocp-tests/scripts/generate-test-entry.sh

```
@@ -0,0 +1,125 @@
+if ! echo "$OCP_VERSION" | grep -qE '^[0-9]+\.[0-9]+$'; then
+  echo "ERROR: Version must be in X.Y format (e.g., 4.22), got: ${OCP_VERSION}" >&2
+  exit 1
+fi
+
+if ! command -v yq &>/dev/null; then
+  echo "ERROR: yq (v4+) is required but not found" >&2
+  exit 1
+fi
+
+# Derive config file path
+dir_parts=(${CI_CONFIG_DIR//\// })
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)